scala/scala-impl/src/org/jetbrains/plugins/hydra/settings/HydraApplicationSettings.scala

```diff
@@ -57,6 +66,12 @@ class HydraApplicationSettingsState {
   @BeanProperty
   var globalArtifactPaths: java.util.Map[String, java.util.List[String]] = new java.util.HashMap()
 
+  @BeanProperty
+  var hydraRepositoryUrl: String = HydraApplicationSettings.DefaultHydraRepositoryUrl
```

Comment:
Can this have type `URL`? - not that `URL` implements `Serializable`, if that matters.